ansible/inventory/submitty

```diff
@@ -0,0 +1,2 @@
+[submitty]
+172.27.23.13
```

Comment:
Add comment about how this is an example inventory and make it point to local host (127.0.0.1).
Also add a new line to the end of the file.